### PR TITLE
HOLD: releases iOS landing page link - DO NOT MERGE UNTIL IOS APP IS READY

### PR DESCRIPTION
**Pull request by owocki**

### CODE REVIEW

**Comment by mbeacom** [comment]
Added the `dont merge` label to this.

**Comment by owocki** [comment]
ios app is ready to go!

**Comment by owocki** [comment]
we need to decide when to pus hthis all out!

**Comment by thelostone-mc** [comment]
@owocki 
Refactored + Cleaned up code + Ensured responsive design across devices 

screen shot 2018-06-13 at 4 59 39 pm

File: app/retail/templates/shared/bottom_notification.html [hunk 1]
```diff
@@ -17,5 +17,5 @@
 {% endcomment %}
 {% load i18n %}
 <div class="row bottom_notification font-caption">
-  {% blocktrans %}<strong>2018 June 8</strong> - Have you seen our  <a style="margin-right: 5px;" href="https://medium.com/gitcoin/tutorial-how-to-price-work-on-gitcoin-49bafcdd201e" target="_blank" rel="noopener noreferrer">Tutorial on How to Price Work on Gitcoin</a>?{% endblocktrans %}
+  {% blocktrans %}<strong>2018 June 15</strong> -{% trans "Browse funded work in OSS on the go.  Check out the " %} <a style="margin-right: 5px;" href="https://gitcoin.co/ios">Gitcoin iOS app</a>!{% endblocktrans %}
```
Comment (owocki) [inline app/retail/templates/shared/bottom_notification.html]: > django.template.exceptions.TemplateSyntaxError: 'blocktrans' doesn't allow other block tags (seen 'trans "Browse funded work in OSS on the go.  Check out the "') inside it

@thelostone-mc

**Comment by thelostone-mc** [comment]
derp derp ! forgot to remove that as per the original design :P

**Comment by thelostone-mc** [comment]
x1

File: app/retail/templates/shared/bottom_notification.html (hunk 1 above)
Comment (thelostone-mc) [reply app/retail/templates/shared/bottom_notification.html]: derp derp! happened during the rebase 
Fixed

**Pull request merged**